### ISSUE TITLE
Postgre Parameters

### ISSUE THREAD

**Issue by funky81** [lwolf/stolon-chart#25]
Hi, Thanks for the great postgre charts....
Just wondering, is it possible to change postgre parameters that contain in stolon configmap annotations?
I want to change the maximum connection paramters.

Thanks

**Comment by lwolf** [comment]
Hi,
Yes it is possible, but not using configmap for now. Please check the stolon docs for details on how you can do it.
https://github.com/sorintlab/stolon/blob/master/doc/cluster_spec.md
https://github.com/sorintlab/stolon/blob/master/doc/postgres_parameters.md

**Comment by funky81** [comment]
Thanks for your info.
Any plans to convert these values into configmap data?
Meanwhile, I'll take route to change annotations via command line yaml (ex:
kubectl apply -f stolon-configmap.yaml)

I haven't really understand how stolonctl works.

**Comment by lwolf** [comment]
Yes, having it configurable at least during cluster creating will be useful.
Regarding `stolonctl` - if you're going to use stolon in production, I suggest investing some time in getting to know it.
